src/mono/sample/wasm/browser-eventpipe/Program.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace Sample
+{
+    public class Test
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine ("Hello, World!");
```

Comment:
Is it intended that nothing actually happens when you run this program?

Reply:
As mentioned in the main.js review, maybe put a single-line comment here to indicate that the real stuff happens in main.js

Reply:
This is some kind of weird Wasm sample build situation: Roslyn needs me to have a `Main` because I don't have `OutputType` specified in the .csproj.  but if I set `OutputType` to `Library` and make Roslyn happy, publishing gets upset and the app doesn't build.

But `Main` never actually gets called.